src/media/css/buttons.styl

```diff
@@ -93,9 +93,10 @@ a.button {
     }
 }
 
-.arrow-button-left,
-.arrow-button-right {
+.arrow-button {
     background-color: $action-positive;
+    background-image: url(../img/pretty/arrow-l.svg);
```

Comment:
I like the scale flip, but I prefer the previous class names. ```.prev``` and ```.next``` are too general.